### ISSUE TITLE
Need version 0.0.19 published for ezno-checker

### ISSUE THREAD

**Issue by andymac4182** [kaleidawave/ezno#117]
From what I can see it appears 0.0.14 is the latest for this crate? https://docs.rs/ezno-checker/latest/ezno_checker/
Is this right?

**Comment by kaleidawave** [comment]
Yes 😅 this repository contains many crates/packages: `ezno-parser` which turns source text into a structures, `ezno-checker` which takes structures and does type checking and semantic analysis and the main `ezno` crate which bundles them into a CLI and library. (as well as a few more internal helper crates)

At the moment each crate has a different version as they have been developed and released independently. 

![image](https://github.com/kaleidawave/ezno/assets/26967284/087bc445-2416-4aa1-95d6-6ec2f959e06a)
> From commit https://github.com/kaleidawave/ezno/commit/b74088ec6fed4e95d2552616159ef2877ec54ad9

You can see what the latest versions of crates through the repository tags: https://github.com/kaleidawave/ezno/tags

Do you think it be clearer if every crate had the same version?

**Comment by andymac4182** [comment]
I think its okay if they were separate. I pulled `ezno` thinking it was similar to `oxc` that exported the various other crates in 1 cohesive crate. I realised later that its the actual CLI checker not a library.
